Cql/CodeGeneration.NET/ExpressionConverter.cs

```diff
@@ -77,7 +72,7 @@ public string ConvertExpression(int indent, Expression expression, bool leadingI
             }
         }
 
-        public string LibraryName { get; }
+        public string LibraryName { get; } = libraryName;
```

Comment:
All assignments to be placed inside the constructor

Reply:
This class has no constructor, as it does not need one...